Source/m_menu.c

```diff
@@ -6120,7 +6146,39 @@ void M_ResetMenu(void)
 
 void M_ResetSetupMenu(void)
 {
-  SetupMenu[0].status = (demo_version < 203) ? 0 : 1;
+  SetupMenu[set_compat].status = (demo_version < 203) ? 0 : 1;
```

Comment:
Sorry, but could you please integrate it into the other checks?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/195/commits/facc1ac00d539112f617272ef956122c7d1635f0